object-storage-oracle-cloud/src/main/java/io/micronaut/objectstorage/oraclecloud/OracleCloudStorageEntry.java

```diff
@@ -54,4 +58,20 @@ public InputStream getInputStream() {
     public GetObjectResponse getNativeEntry() {
         return objectResponse;
     }
+
+    @NonNull
+    @Override
+    public Map<String, String> getMetadata() {
+        if (CollectionUtils.isNotEmpty(objectResponse.getOpcMeta())) {
+            return objectResponse.getOpcMeta();
```

Comment:
Is the map returned here immutable? If not wrap in `Collections.unmodifiableMap`